src/aleph/vm/orchestrator/run.py

```diff
@@ -105,6 +105,10 @@ async def create_vm_execution_or_raise_http_error(vm_hash: ItemHash, pool: VmPoo
         logger.exception(error)
         pool.forget_vm(vm_hash=vm_hash)
         raise HTTPInternalServerError(reason="Host did not respond to ping")
+    except Exception as error:
+        logger.exception(error)
+        pool.forget_vm(vm_hash=vm_hash)
+        raise HTTPInternalServerError(reason="unhandled error during initialisation")
```

Comment:
```suggestion
        raise HTTPInternalServerError(reason="unhandled error during initialisation") from error
```
Related to https://github.com/aleph-im/aleph-vm/pull/450